docs/content/tutorial.fsx

```diff
@@ -1,19 +1,134 @@
 (*** hide ***)
 // This block of code is omitted in the generated HTML documentation. Use 
 // it to define helpers that you do not want to show in the documentation.
-#I "../../bin"
+#I "../../bin/kafunk/"
 
 (**
-Introducing your project
-========================
+Kafunk - F# Kafka client
+======================
 
-Say more
+Set up Zookeeper and Kafka
+--------------------------
 
+The [Apache Kafka documentation](http://kafka.apache.org/documentation.html) contains an excellent [Quick Start guide](http://kafka.apache.org/documentation.html#quickstart) to set up Kafka.
+For this tutorial we will use the default parameters for Zookeeper and Kafka which means we will connect against localhost and port 9092.
+
+Connecting to Kafka and retrieving meta data
+--------------------------------------------
+
+This example demonstrates a few uses Kafunk. 
+We start by referencing the Kafunk F# Kafka client and  ceate a connection to a host:
 *)
+
 #r "kafunk.dll"
 open Kafunk
 
-Library.hello 0
+let conn = Kafka.connHost "localhost"
```

Comment:
Localhost is a good call. We put too many Franz Kafka references in the code. :+1: